tests/phpunit/tests/rest-api/rest-pattern-directory-controller.php

```diff
@@ -461,4 +551,21 @@ static function ( $return, $args, $url ) use ( $blocked_host ) {
 			3
 		);
 	}
+
+	/**
+	 * Returns a reflection method and changes its scope.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param $object An object that has a private method that has to be called.
+	 * @param $method Name of the method.
+	 *
+	 * @return ReflectionMethod
+	 */
+	private function get_reflection_method( $object, $method ) {
```

Comment:
Tests do not need these kind of comments.

Reply:
Good to know.
Fixed in 7d1c51221c42336c151aa454c7ce3b407b7e250f